docs_assets/pages/3.1_metastore.md

```diff
@@ -24,3 +24,21 @@ To change the schema being used, copy the contents of the `schema` directory fro
 
 @warning
   Warning: The schema is actively used by the catalog to verify the validity of the data. Making changes to the schema after data is present in the catalog should be done with care as non-backward-compatible changes to the schema could cause issues. Look at Drupal::metastore::SchemaRetriever::findSchemaDirectory() for context.
+
+## Data Dictionaries
```

Comment:
@clayliddell Will all three options require that the **Datastore MySQL Import** module be enabled?

Reply:
No, the datastore mysql import module is not required.